ip/draft-ietf-detnet-ip.xml

```diff
@@ -357,46 +363,48 @@
       </t>
 
 
-      <section title="End-System Specific Considerations">
+      <section title="End-system-specific Considerations">
         <t>
-          Data-flows requiring DetNet service are generated and terminated on
-          end systems.  This document deals only with IP end systems.  The
-          protocols used by an IP end system are specific to an application
-          and end systems peer with end systems using the same application
-          encapsulation format.  This said, DetNet's use of 6-tuple IP flow
+          IP Data-flows requiring DetNet service are generated and terminated 
+		  on IP end systems.
+		  Encapsulation of application data on the IP end system is application 
+		  specific and it has to match the encapsulation format of the peer IP
```

Comment:
Who is "it" in this sentence?  Also this document can't place requirements on non-detnet components.  If the change is being made for editorial reasons, I suggest using the original text -- if not, let's discuss on list (perhaps as part of last call.

Reply:
This sentence was intended as editorial clarification based on Ethan's comment. I have created this sentence, but as You commented it seems not to provide the required clarification. The original was not clear enough as well. I would suggest to solve this during in the last call. Alternatively please propose a new sentence with better phrasing.